tests/components/number/test_device_action.py

```diff
@@ -0,0 +1,128 @@
+"""The tests for Humidifier device actions."""
```

Comment:
Tests for Humidifier? I think you mean Number?

Reply:
Oops, based the tests on my another PR for humidifier integration, thanks!